src/verifai/samplers/feature_sampler.py

```diff
@@ -163,15 +193,41 @@ def __init__(self, space, makeLengthSampler, makeDomainSampler):
 
     def nextSample(self, feedback=None):
         if self.lengthSampler is None:
```

Comment:
As above, this should not be necessary -- defining `getSample` and `update` should be enough.